.packit.yml

```diff
@@ -90,9 +90,9 @@ jobs:
     trigger: pull_request
     identifier: "unit/rhel"
     targets:
-      rhel-9-x86_64:
+      centos-stream-10-x86_64: # TODO: Change me back to RHEL
```

Comment:
Does this need to be changed to RHEL before we merge?

Reply:
This needs to be changed to RHEL whenever copr adds rhel-10 bulding

Reply:
Ok. We'll leave it on centos-stream-10 then. Thanks!